wca-guidelines.md

```diff
@@ -137,8 +137,11 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-11><incidents><incidents> Article 11: Incidents
 
-- 11e+) [CLARIFICATION] Since an appeal is not guaranteed to be successful, the competitor may choose to keep the timer running while appealing it, and resume the attempt when appropriate.
+- 11e+) [CLARIFICATION] Since an appeal is not guaranteed to be successful, the competitor may choose to keep the timer running while appealing it, and resume the solve when appropriate.
 - 11e++) [RECOMMENDATION] The WCA Delegate should record the reason(s) for the extra attempt (e.g. on the back of the score sheet).
+- 11e+++) [RECOMMENDATION] The WCA Delegate should use stronger discretion when granting an extra attempt after the original attempt is finished. The extra attempt should be granted on very exceptional circumstances.
```

Comment:
```suggestion
- 11e+++) [RECOMMENDATION] The WCA Delegate should use stronger discretion when granting an extra attempt after the original solve is finished. The extra attempt should be granted on very exceptional circumstances.
```

Should've noticed earlier, but still. Explanation: the regulation itself is about granting extras when the competitor is appealing before finishing the attempt, current wording doesn't make sense at all; the original wording with the `solve` I suggested internally is to prevent a situation like [this](https://www.worldcubeassociation.org/incidents/69).

Reply:
```suggestion
- 11e+++) [RECOMMENDATION] The WCA Delegate should use stronger discretion when granting an extra attempt after the solve is finished. The extra attempt should be granted only in exceptional circumstances.
```
- attempts not solves are replace by extras so 'original' doesn't seem meaningful in this context
- update for clearer English
- 'very exceptional' suggests an extremely high burden for granting such an extra that is at odds with the first sentence. I believe merely 'exceptional' is sufficient for the stress that needs to be granted here